Documentation/En/Index.xyl

```diff
@@ -14,7 +14,7 @@
 
   <h2 id="Introduction" for="main-toc">Introduction</h2>
 
-  <p>Software quality is a vast topic in Computer Science. There is a lot of
+  <p>Software quality is a vast topic in Computer Science. There are a lot of
```

Comment:
I think `are` should be used for plural forms, e.g. `there is a lot of rain`, `there are a lot of penguins`.